lib/auth.ts

```diff
@@ -179,13 +190,15 @@ export class Auth {
   }
 
   get expired() {
-    return Date.now() > this.data.expires;
+    return !!this.data.refresh_token && Date.now() > this.data.expires;
```

Comment:
Expiration is only bound to the access token. Existence of an refresh token does not impact it.